app/services/StubIdentityService.scala

```diff
@@ -0,0 +1,24 @@
+package services
+
+import cats.data.EitherT
+import com.gu.identity.play.{IdMinimalUser, IdUser, PrivateFields, PublicFields}
+import monitoring.SafeLogger
+import play.api.mvc.RequestHeader
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class StubIdentityService extends IdentityServiceOrStub {
+  def getUser(user: IdMinimalUser)(implicit req: RequestHeader, ec: ExecutionContext): EitherT[Future, String, IdUser] = {
+    val privateFields = PrivateFields(firstName = Some("Frosty"), secondName = Some("The Snowman"))
+    val stubTestUser: IdUser = IdUser("123456", "nonsense@gu.com", PublicFields(None), Some(privateFields), None)
+
+    SafeLogger.info(s"Stubbed identity service active. Returning test names $privateFields")
+
+    EitherT(Future.successful(Either.cond(true, stubTestUser, "This stub should always give a right")))
```

Comment:
Interesting, I haven't seen `Either.cond` before.  But can you not just do `Future.successful(Right(stubTestUser))`

Reply:
I used the `Either.cond` because I need the return type to be `EitherT[Future, String, IdUser]` 

I think `Future.successful(Right(stubTestUser))` is a `EitherT[Future, Nothing, IdUser]`